icon/types/index.ts

```diff
@@ -0,0 +1,8 @@
+import { SVGProps } from 'react';
+
+export type Sizes = 's' | 'm' | 'l' | 'xl';
+
+export type PickedSVGProps = Pick<
+  SVGProps<SVGPathElement>,
+  'fill' | 'stroke' | 'strokeLinecap' | 'strokeLinejoin' | 'strokeWidth' | 'transform' | 'd'
+>;
```

Comment:
Can these not live closer to the usage? not in a types.ts file?

Reply:
Sizes will need to live at this point because it is used both in Icon.tsx and the icon specialization IconProps type.
I have now moved PickedSVGProps to Icon.tsx though.
[https://github.com/deanlot/test-backlight/pull/13/commits/9fc07d42dcd664f4b193a3091606b24c1ebb83aa](url)